doc/interop-test-descriptions.md

```diff
@@ -1011,16 +1011,49 @@ languages. Therefore they are not part of our interop matrix.
 #### rpc_soak
 
 The client performs many large_unary RPCs in sequence over the same channel.
-The number of RPCs is configured by the experimental flag, `soak_iterations`.
+
+This test is configurable via a few different command line flags:
+
+* `soak_iterations`: controls the number of RPCs to perform.
+
+* `soak_max_failures`: an inclusive upper limit on the number of RPC failures
+  that should be tolerated (i.e. after which the test process should
+  still exit 0). A failure is considered to be either a non-OK status or an RPC
+  whose latency exceeds `soak_per_iteration_max_acceptable_latency_ms`.
+
+* `soak_per_iteration_max_acceptable_latency_ms`: an inclusive upper limit
```

Comment:
"inclusive" is a bit confusing here. Strictly speaking that could mean adding just under 1ms to the value. I assume that is being said this way to allow using a ms-precise timer. But in that case there's actually 1ms worth of jitter. It seems much more clear to just remove "inclusive" and require implementations to use a sub-ms-precision timer. Using a ms-precise timer "naturally" be done with the wall-clock time in your language, which is wrong for durations.

Reply:
changed wording to just require a sub millisecond precision timer